transport-native-kqueue/pom.xml

```diff
@@ -79,6 +79,8 @@
                   <configureArgs>
                     <arg>${jni.compiler.args.ldflags}</arg>
                     <arg>${jni.compiler.args.cflags}</arg>
+                    <!-- weak_import was added in 10.2 so ensure we explicitly set the artifact -->
+                    <arg>MACOSX_DEPLOYMENT_TARGET=10.2</arg>
```

Comment:
@Scottmitch in the comment you say `weak_import` but we not use this in the ldflags. Are you refer to `-z now` ? Can you clarify this ?